gravity-forms/gw-require-unique-values.php

```diff
@@ -130,6 +131,12 @@ public function get_filtered_value( $field ) {
 
 	public function get_value_hash( $value ) {
 
+		if ( ! $this->_args['case_sensitive'] ) {
+			foreach( $value as &$v ) {
+				$v = strtolower( $v );
+			}
```

Comment:
FWIW, this can be simplified using `array_map`:

```suggestion
			$value = array_map( 'strtolower', $value );
```